.github/workflows/pr-env-deploy.yml

```diff
@@ -78,6 +78,22 @@ jobs:
       - name: Build Project
         run: make build
 
+      - name: Configure Management Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: eu-west-2
+          role-to-assume: ${{ secrets.MGMT_ROLE_ARN }}
+          role-session-name: github-actions-ci-${{ needs.set-environment-id.outputs.environment_id }}
+
+      - name: Setup Python environment
```

Comment:
Do we need to run the "Setup Python environment" step again? It's already run earlier in this action :point_up:

Reply:
yeah copy paste has exposed me 😂